src/bp/ui-studio/src/web/index.jsx

```diff
@@ -45,12 +46,14 @@ if (!window.BOT_ID) {
   // Do not use "import App from ..." as hoisting will screw up styling
   const App = require('./components/App').default
 
-  ReactDOM.render(
-    <Provider store={store}>
-      <HotKeys keyMap={keyMap}>
-        <App />
-      </HotKeys>
-    </Provider>,
-    document.getElementById('app')
-  )
+  initializeTranslations().then(() => {
+    ReactDOM.render(
+      <Provider store={store}>
+        <HotKeys keyMap={keyMap}>
+          <App />
+        </HotKeys>
+      </Provider>,
+      document.getElementById('app')
+    )
+  })
```

Comment:
Same problem for studio. Maybe the call to initializeTranslations should be moved somewhere else